VectorRumble.Shared/Gameplay/Ship.cs

```diff
@@ -231,20 +227,28 @@ class Ship : Actor
         /// <summary>
         /// Timer for how long the player has been spawned for, to fade in
         /// </summary>
-        private float fadeInTimer = 0f;
+        private float fadeInTimer = 0f;
         #endregion
-
+
         #region Properties
+        [ContentSerializerIgnore]

```

Comment:
do we still need all these [Content*] related attributes?
and the `using Microsoft.Xna.Framework.Content`?

we are not using the content intermediate serialiser anymore afaik.

Reply:
No.